src/DependencyInjection/Configuration.php

```diff
@@ -176,6 +170,7 @@ public function getConfigTreeBuilder()
                         // NEXT_MAJOR : remove this option
                         ->booleanNode('legacy_twig_text_extension')
                             ->info('Use text filters from "twig/extensions" instead of those provided by "twig/string-extra".')
+                            ->setDeprecated('The child node "%node%" at path "%path%" is deprecated and will be remove in sonata-project/admin-bundle 4.0.')
```

Comment:
```suggestion
                            ->setDeprecated('The child node "%node%" at path "%path%" is deprecated since sonata-project/admin-bundle 3.x and will be remove in 4.0.')
```